e2e-scripts/sippy-e2e-sippy-e2e-test-commands.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# In Prow CI, SIPPY_IMAGE variable is defined in the sippy-e2e-ref.yaml file as a
+# dependency so that the pipeline:sippy image (containing the sippy binary)
+# will be available to start the sippy-load and sippy-server pods.
+# When running locally, the user has to define SIPPY_IMAGE.
+echo "The sippy CI image: ${SIPPY_IMAGE}"
+
+# If you're using Openshift, we use oc, if you're using plain Kubernetes,
+# we use kubectl.
+#
+KUBECTL_CMD="${KUBECTL_CMD:=oc}"
+echo "The kubectl command is: ${KUBECTL_CMD}"
+
+# Launch the sippy api server pod.
+cat << END | ${KUBECTL_CMD} apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sippy-server
+  namespace: postgres
+  labels:
+    app: sippy-server
+spec:
+  containers:
+  - name: sippy-server
+    image: ${SIPPY_IMAGE}
+    imagePullPolicy: Always
+    ports:
+    - name: www
+      containerPort: 8080
+      protocol: TCP
+    - name: metrics
+      containerPort: 12112
+      protocol: TCP
+    readinessProbe:
+      exec:
+        command:
+        - echo
+        - "Wait for a short time"
+      initialDelaySeconds: 10
+    resources:
+      limits:
+        memory: 2G
+    terminationMessagePath: /dev/termination-log
+    terminationMessagePolicy: File
+    command:
+    - /bin/sippy
+    args:
+    - --server
+    - --listen
+    - ":8080"
+    - --listen-metrics
+    -  ":12112"
+    - --local-data
+    -  /opt/sippy-testdata
+    - --database-dsn=postgresql://postgres:password@postgres.postgres.svc.cluster.local:5432/postgres
+    - --log-level
+    - debug
+    - --mode
+    - ocp
+  imagePullSecrets:
+  - name: regcred
+  dnsPolicy: ClusterFirst
+  restartPolicy: Always
+  schedulerName: default-scheduler
+  securityContext: {}
+  terminationGracePeriodSeconds: 30
+END
+
+# The basic readiness probe will give us at least 10 seconds before declaring the pod as ready.
+echo "Waiting for sippy api server pod to be Ready ..."
+${KUBECTL_CMD} -n postgres wait --for=condition=Ready pod/sippy-server --timeout=30s
+
+is_ready=0
+for i in `seq 1 20`; do
+  c=$(${KUBECTL_CMD} -n postgres logs sippy-server|tail -1|grep "Refresh complete"|wc -l)
```

Comment:
There's a problem here, tail -1 will fail often if *anything* gets logged after the refresh complete, which just happened here. I think you just need to remove the tail command entirely and it should work. This could use refinement though.